WalletWasabi/WabiSabi/Backend/Rounds/Round.cs

```diff
@@ -30,6 +30,8 @@ public Round(RoundParameters roundParameters)
 			Coinjoin = Transaction.Create(Network);
 
 			Hash = new(HashHelpers.GenerateSha256Hash($"{Id}{MaxInputCountByAlice}{MinRegistrableAmount}{MaxRegistrableAmount}{RegistrableWeightCredentials}{AmountCredentialIssuerParameters}{WeightCredentialIssuerParameters}{FeeRate.SatoshiPerByte}"));
+
+			RemainingWeightUnits = 399_818; //TODO: move this to WabiSabiConfig.
```

Comment:
the `TODO` comment applies just to the default value, right? not the entire property?

Reply:
Yes